#84: Anonymous feature tracking (#85)
exasol/oracle-virtual-schema: doc/changes/changes_5.0.0.md

# Oracle Virtual Schema 5.0.0, released 2026-04-??

Code name: Anonymous telemetry

## Summary

This release adds anonymous feature-usage telemetry via `telemetry-java`. See the [documentation](https://github.com/exasol/telemetry-java/blob/main/doc/app-user-guide.md) for details on collected data and opt-out behavior.

## Breaking Change

> Starting with this release, the Oracle Virtual Schema does not support Exasol versions 7.1 any more. Only LTS version 2025.1.x and the current version are supported.

Reviewer: ```suggestion
Starting with this release, the Oracle Virtual Schema does not support Exasol versions 7.1 anymore. Only LTS version 2025.1.x and the current version are supported.
```
